### DISCUSSION TITLE
suggestion: system notification when a command complete

### DISCUSSION THREAD

**Discussion by sylvestre** [uutils/coreutils#4875] [Ideas]
For example, we could send an operating system notification when cp ends (or other long command)

For example
cp -R --notification source dest

For example, using:
https://crates.io/crates/notify-rust

**Comment by tertsdiepraam** [comment]
Sounds interesting. I think some terminal emulators do this automatically already (at least KDE's konsole does), so maybe it's not really a priority?

**Reply by sylvestre** [reply]
Clearly not a priority, i didn't know that kde was doing :)
